src/Features/Core/Portable/CodeFixes/FixAllOccurrences/FixSomeCodeAction.cs

```diff
@@ -16,7 +16,7 @@ internal abstract class FixSomeCodeAction : CodeAction
         private static readonly HashSet<string> s_predefinedCodeFixProviderNames = GetPredefinedCodeFixProviderNames();
 
         internal readonly FixAllState FixAllState;
-        private readonly bool _showPreviewChangesDialog;
+        private bool _showPreviewChangesDialog;
```

Comment:
Was tehre a bug in the 'make readonly' fixer that caused this?

Reply:
Yes. The commit links to the bug.

Reply:
Got it.  Thanks!